Dev

dipenpradhan opened the pull request:
Crawler implemented

james-jana commented on src/com/dipenpradhan/janacrawler/Crawler.java:
```diff
@@ -0,0 +1,119 @@
+                        String href = ah.attr("abs:href");
+                        boolean ignore = false;
+                        for (String ext : IGNORED_EXTENSIONS) {
+                            if (href.contains(ext)) {
```
This is nested pretty deeply. Do you think it would make sense to abstract some of this into separate functions?

james-jana commented:
Can you include a build script so that this can be compiled from source?

dipenpradhan commented:
Ok, I'll do that and create a new pull request.

james-jana commented:
You should just be able to commit again to the same PR, and it will update.

On Tue, Feb 16, 2016 at 5:17 PM Dipen Pradhan notifications@github.com
wrote:

> Ok, I'll do that and create a new pull request.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/dipenpradhan/email-crawler/pull/1#issuecomment-184897807
> .

dipenpradhan commented:
Ok!

dipenpradhan commented:
Hi @james-jana , I have:
1. Added a gradle buildscript 
2. Updated README.md with instructions for building the project
3. Cleaned up the code and extracted the nested parts into separate functions.
4. Added comments
